app/javascript/chat/RequestManager/PersonalInvitationListItem.jsx

```diff
@@ -0,0 +1,54 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+const RequestListItem = ({ request, updateMembership }) => (
+  <div className="crayons-card mb-6">
+    <div className="crayons-card__body channel-request-card">
+      <div className="request-message d-flex flex-wrap">
+        You got invitation to join <b>{request.chat_channel_name}</b>.
+      </div>
+      <div className="request-actions">
+        <button
+          type="button"
+          className="crayons-btn  crayons-btn--s crayons-btn--danger m-2"
```

Comment:
Can we change these to use the design system buttons as well?

Using `import { Button } from '@crayons'` and this will also allow to remove some of the class names. There's a few of these in this and other files.

Related comment: https://github.com/forem/forem/pull/8945#discussion_r455050177

Reply:
Yes sure I will update